app/controllers/lawyers_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LawyersController < ApplicationController
+  def index
+    @lawyer = Lawyer.all
+  end
```

Comment:
Right now the #index action is rendering nothing, in our case, our app is an API, so the action should render JSON, a good idea would be to add a `render` statement, like this:
```suggestion
  def index
    @lawyers = Lawyer.all
    render json: { lawyers: @lawyers  }
  end
```